functionalTest/wdio.conf.js

```diff
@@ -41,10 +41,11 @@ exports.config = {
   // Define all options that are relevant for the WebdriverIO instance here
   //
   // Level of logging verbosity: trace | debug | info | warn | error | silent
-  logLevel: "info",
+  logLevel: "warn",
   bail: 0,
   // Set a base URL in order to shorten url command calls. If your `url` parameter starts
-  baseUrl: 'https://sta-homebrew-iteam.s3.eu-west-2.amazonaws.com',
+  // baseUrl: 'https://sta-homebrew-iteam.s3.eu-west-2.amazonaws.com',
+  baseUrl: 'http://localhost:3000',
```

Comment:
Note I've set the default to run against a local 'npm start' run of reactDashboard.